backend/src/main/java/com/benine/backend/camera/CameraController.java

```diff
@@ -44,6 +42,22 @@ public void addCamera(Camera camera) {
     highestIdInUse++;
     cameras.add(camera);
   }
+  
```

Comment:
What are these two methods used for? Since I think you could just replace the call to these methods to a call to ServerController.getInstance().getCameraController() right?  but maybe I'm missing something.